paasta_tools/marathon_tools.py

```diff
@@ -689,7 +702,11 @@ def format_marathon_app_dict(self) -> FormattedMarathonAppDict:
             'args': self.get_args(),
         }
 
-        if net == 'BRIDGE':
+        if complete_config['container']['type'] == 'MESOS' and self.get_gpus() is not None:
```

Comment:
we should probably show a warning to the user when containerizer is DOCKER and gpus is set? and the same with containerizer MESOS  and net==BRIDGE?